Source/JavaScriptCore/inspector/protocol/CSS.json

```diff
@@ -274,7 +274,8 @@
             "enum": [
                 "rendered",
                 "flex",
-                "grid"
+                "grid",
+                "event"
```

Comment:
I agree that it doesn't really have anything to do with the idea of "layout" (tho there are things like "event regions" that one could argue as being related to "layout").  I wanted to move this entire thing to be something like `DOM.NodeStateFlag` but that felt a bit overkill for this (not to mention it would involve changing a whole bunch more than just the name of the enum since we'd also probably want to move the events/commands to `DOM` as well).

I don't really feel strongly any way, so I'm happy to do whatever.  I went with this only because it was the least modification.

Reply:
Hmmm... this is probably fine then.